src/components/ui-shell/_platform-nav.scss

```diff
@@ -0,0 +1,146 @@
+@import '../../globals/scss/colors';
+@import '../../globals/scss/css--helpers';
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/vars';
+@import 'functions';
+
+@mixin platform-nav {
+  //----------------------------------------------------------------------------
+  // Platform Navigation
+  //----------------------------------------------------------------------------
+  .#{$prefix}--platform__nav {
+    position: absolute;
+    top: get-fixed-size(6);
+    bottom: 0;
+    left: 0;
+    background-color: $ibm-colors__gray--90;
```

Comment:
Need theme tokens for `$ibm-colors__` refs in this file.

Reply:
We don't have any for this one yet, sadly :/